Posit Infrastructure/Guide to using renv to create reproducible projects.md

```diff
@@ -0,0 +1,47 @@
+# Guide to using [{renv}](https://rstudio.github.io/renv/) to create reproducible projects
+
+## Purpose
+
+Our code should be reproducible and shareable. But sometimes things fail to run properly when we return to them, or on a colleague's machine. The R package [{renv}](https://rstudio.github.io/renv/) ensures that the exact set of packages (and their versions) required to run the code in a specific project are recorded and kept available to current and future users. 
+
+This guide provides basic step-by-step instructions for using [{renv}](https://rstudio.github.io/renv/), as well as more detailed information. 
+
+## Step-by-step instructions
+
+Pre-requisite: a project (preferably version-controlled using git).
+
+#### Setting up [{renv}](https://rstudio.github.io/renv/) for the first time in a project:
+1.	Open the project.
+2.	Call `renv::init()` to initialise renv within that project. This can take a while depending on the size of the project.
+3.	Calling `renv::status()` should confirm the new 'lockfile' is synced with the project (i.e., contains details of all the required packages).
+4.	Commit the changes (additional files and folders) to version control.
+
+#### Adding (/removing) packages to the project:
+1.	Install (or remove) the required packages as usual. 
+2.	Change the code, by adding (or removing) calls to `library()` or `require()`, or the package function calls themselves e.g. `dplyr::group_by()`.
```

Comment:
I assume specifying preferred packages using conflicted() would also work?

Reply:
Hey, @charlie-smith - as far as I can tell, the `{conflicted}` package works on the code level, so as long as the codebase still contains the relevant function calls, it wouldn't be affected by `{renv}`.